### PR TITLE
Adds Create and Edit Template to new sidebar

### PR DESCRIPTION
**Pull request by SaraVieira**
Also fixes the `css` prop yay!

### CODE REVIEW

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ProjectInfo/index.tsx [hunk 1]
```diff
@@ -86,7 +67,7 @@ export const ProjectInfoComponent = ({ theme }) => {
               gap={2}
               align="center"
               marginBottom={4}
-              style={{ padding: `0 ${theme.space[3]}px` }}
+              css={css({ paddingX: 2 })}
```
Comment (siddharthkp) [inline packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ProjectInfo/index.tsx]: the future is here!

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/3358/commits/4bb3c1a3b7a9f1980a6b2420e22c0c28d8fae056 is at https://pr3358.build.csb.dev/s/new.

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ProjectInfo/Config.tsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+      <Stack
+        justify="center"
+        css={css({
+          bottom: 3,
+          position: 'absolute',
+          width: '100%',
+        })}
+      >
+        <Button
+          // @ts-ignore
```
Comment (siddharthkp) [inline packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ProjectInfo/Config.tsx]: why does ts hate this?

Comment (SaraVieira) [reply packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ProjectInfo/Config.tsx]: Because its a styled div :(

Comment (siddharthkp) [reply packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ProjectInfo/Config.tsx]: wait, styled div or button 😱

**Pull request merged**